### ISSUE TITLE
Extension causes form inputs/textareas to contain HTML for click2dial icon when matching "phone numbers"

### ISSUE THREAD

**Issue by lukebeer** [BroadSoft-Xtended/Project-Chrome-Extension#4]
When editing a html textarea, in my example a webmin zone file. The extension matches the serial number and inserts html for click2dial on submit in turn, screwing up the zone file.

An exclusion for inputs/textareas needs to be implemented for number matching.

Example here - http://jsfiddle.net/5Weya/

**Comment by broadsoftxtended** [comment]
Yep,  in addition the regular expressions in contentscript.js needs refining; it's a little over eager right now. e.g. IP addresses are matched as phone numbers.

You can turn off the click-to-dial feature via the preferences page.
